tracer/build/PackageVersionsGeneratorDefinitions.json

```
@@ -597,5 +597,23 @@
       "1.0.1",
       "1.*.*"
     ]
+  },
+  {
+    "IntegrationName": "Yarp",
+    "SampleProjectName": "Samples.Yarp.DistributedTracing",
+    "NugetPackageSearchName": "Yarp.ReverseProxy",
+    "MinVersion": "1.0.0",
+    "MaxVersionExclusive": "2.2.0",
```

I think this should be 3.0.0 so we automatically test future versions?
```suggestion
    "MaxVersionExclusive": "3.0.0",
```

---

Fixed in b59e6ef55c3e420920cc987aaa958666caffcb4b